ledger/participant-integration-api/src/main/scala/platform/indexer/parallel/UpdateToDBDTOV1.scala

```diff
@@ -212,13 +205,7 @@ object UpdateToDBDTOV1 {
                   if (exercise.consuming) exercise.stakeholders.map(_.toString) else Set.empty,
                 tree_event_witnesses =
                   blinding.disclosure.getOrElse(nodeId, Set.empty).map(_.toString),
-                create_argument = None,
-                create_signatories = None,
-                create_observers = None,
-                create_agreement_text = None,
-                create_key_value = createKeyValue
-                  .map(compressionStrategy.createKeyValueCompression.compress),
-                create_key_hash = None,
+                // TODO append-only: createKeyValue is not stored, check whether we need it for upcoming ledger API additions
```

Comment:
Yes, AFAIR @tudor-da 's mutable state cache require the create_key_hash to be stored for consuming exercises, so that key-unassignments can be tracked. 

@tudor-da : What else do you need?

@rautenrieth-da : AFAIS, we can add the createKeyValue vai a (potentially costly) migration at a later stage when implementing Ledger API features. Right now, I'd focus on what we need for the current Ledger API and the 1k trade/s.

Reply:
We need the `create_key_value` and we've already implemented it.

Reply:
Fixed in this PR.